code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -102,7 +102,7 @@
 	icon_state = "alienpistol"
 	item_state = "alienpistol"
 	fire_delay = 10 // Handguns should be inferior to two-handed weapons. Even alien ones I suppose.
-	charge_cost = 480 // Five shots.
+	charge_cost = 240 // Five shots.
```

Comment:
```suggestion
	charge_cost = 240 // Ten shots.
```
Update comments to reflect things accurately when you change a number - it'd be appreciated.